src/dproto_tcp.erl

```diff
@@ -25,8 +25,12 @@
 -export_type([ttl/0, bucket_info/0, tcp_message/0,
               batch_message/0, stream_message/0]).
 
+
 -type ttl() :: pos_integer() | infinity.
 
+% Read repair option: 0 = off, 1 = on, 2 = default
+-type rr_opt() :: 0 | 1 | 2.
```

Comment:
Exposing those as atoms would be a lot better. The wire representation (number) odes not have to meet the API representation.

This would lead to more readable code.

i.e. `{rr, on}` instead of `{rr, off}`

also by decuppling representation form API it allows to change representation if required.